src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -434,11 +490,35 @@
         return totalsMatchIfRequired;
     },
 
-    /******************************** Edit Batch Modal Functions *****************************/
-
     /**
-     * @description: checks that user has all necessary permissions and then launches modal or displays error
+     * @description: check if there are conflicting gifts in the table.
+     * @return: boolean indicating if there is a conflict
      */
+    tableHasConflictingGifts: function(component) {
+        const rows = component.get('v.data');
+        const labels = component.get('v.labels');
+
+        let conflictFound = false;
+
+        let donationIds = rows.reduce(function (filtered, row) {
+            let opportunityLookup = row[labels.opportunityImportedLookupField];
+            if (opportunityLookup != undefined) {
+                filtered.push(opportunityLookup);
+            }
+            return filtered;
+        }, []);
+
+        conflictFound = donationIds.some(function (donationId, index) {
```

Comment:
@ptewson-sfdo @kenslewis - What do ya'll think about the cost of having two 'loops' here (some and indexOf)? Acceptable?